superset/connectors/sqla/models.py

```diff
@@ -99,21 +104,20 @@ class TableColumn(Model, BaseColumn):
         s for s in export_fields if s not in ('table_id',)]
     export_parent = 'table'
 
-    @property
-    def sqla_col(self):
-        name = self.column_name
+    def sqla_col(self, db_engine_spec, label=None):
```

Comment:
I think `db_engine_spec` is available through `self.table.database.db_engine_spec`, so no need to receive it as a method argument.

Reply:
Whoops; `db_engine_spec` removed from all methods, as it is already available in all classes. I also renamed the method to `get_sqla_col` from just `sqla_col`, as is was previously a property.